python/triton/runtime/autotuner.py

```diff
@@ -153,6 +153,15 @@ def run(self, *args, **kwargs):
                 timings = {config: self._bench(*args, config=config, **kwargs) for config in pruned_configs}
                 bench_end = time.time()
                 self.bench_time = bench_end - bench_start
+
+                if os.getenv("TRITON_PRINT_AUTOTUNING_ALL", None) == "1":
+                    print(
+                        f'\nPrinting ALL Multiple Triton autotuning Configs with timings in sorted order for kernel {self.fn}:'
+                    )
+                    sorted_configs = builtins.sorted(timings, key=timings.get)
+                    for config in sorted_configs:
+                        print(f'Triton autotune config: [{config}]; Triton autotune timing: {timings[config]}')
+
                 self.cache[key] = builtins.min(timings, key=timings.get)
                 self.pre_hook(args, reset_only=True)
```

Comment:
@Jokeren do you think it's reasonable to pass `timings` here? Then we can leverage `pre_hook` to add this behavior

Reply:
It will be confusing. The name of this hook is `pre_hook` but not `post_hook`

Reply:
timing info is already stored in the autotuner. So if we just go use `kernel_name.configs_timings`, we can obtain the info anyway